scenarios/python/console/premium_speech_demo/premium_speech_demo.py

```diff
@@ -0,0 +1,85 @@
+import os
+import azure.cognitiveservices.speech as speechsdk
```

Comment:
Please use a more meaningful name for the folder instead of `premium_speech_demo/` and `premium_speech_demo.py`.

E.g. Use `rewriting` as folder name and `app.py` as main python file name.